src/screens/reader/components/Hooks/useTextModifications.ts

```
@@ -0,0 +1,125 @@
+
+  const html = useMemo(() => {
+    let chText = chapterText;
+    const validFlags = new Set(['g', 'm', 'i', 'y', 'u', 'v', 's', 'd']);
+    readerSettings.removeText.forEach(text => {
+      // test if text is regex
+      const m = text.match(/^\/(.*)\/([gmiyuvsd]*)$/);
+      if (m) {
+        const flags = m[2] ?? '';
+        const hasInvalidFlags = [...flags].some(f => !validFlags.has(f));
+        if (hasInvalidFlags) {
+          console.warn('Invalid regex flags in removeText:', text);
```

ESLint

[error] Unexpected console statement.

(no-console)